.github/workflows/pre-release.yml

```diff
@@ -363,35 +374,38 @@ jobs:
           enforce_admins: true
 
       - name: Create GitHub Pre-Release
+        working-directory: keptn
         env:
           GITHUB_TOKEN: ${{ secrets.KEPTN_BOT_TOKEN }}
           RELEASE_TAG: ${{ steps.create-release-package.outputs.tag-name }}
         run: |
           gh release create "$RELEASE_TAG" --draft --prerelease --notes-file "${{ env.RELEASE_NOTES_FILE }}" --title "$RELEASE_TAG"
 
       - name: Upload Release Assets
+        working-directory: keptn
         env:
           GITHUB_TOKEN: ${{ secrets.KEPTN_BOT_TOKEN }}
           RELEASE_TAG: ${{ steps.create-release-package.outputs.tag-name }}
         run: |
-          gh release upload "$RELEASE_TAG" ./dist/keptn-installer/*.tgz
-          gh release upload "$RELEASE_TAG" ./dist/keptn-cli/*.tar.gz
+          gh release upload "$RELEASE_TAG" ../dist/keptn-installer/*.tgz
+          gh release upload "$RELEASE_TAG" ../dist/keptn-cli/*.tar.gz
 
-      - name: Upload Helm Charts to Google Cloud
-        id: upload_helm_charts
+      - name: Install Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.7.2
+
+      - name: Upload helm charts
         env:
-          GCLOUD_SERVICE_KEY: ${{ secrets.GCLOUD_SERVICE_KEY }}
-          GCLOUD_PROJECT_NAME: ${{ secrets.GCLOUD_PROJECT_NAME }}
-          CLOUDSDK_COMPUTE_ZONE: "us-east1-b"
-          CLOUDSDK_REGION: "us-east1"
+          ARTIFACTS: "./dist/keptn-installer/*.tgz"
+          RELEASE_TAG: ${{ steps.create-release-package.outputs.tag-name }}
         run: |
-          echo "Installing GCloud CLI"
-          export OS_TYPE="linux"
-          mkdir ~/downloads
-          ./test/utils/download_and_install_gcloud.sh
-          echo ${GCLOUD_SERVICE_KEY} | base64 --decode > ~/gcloud-service-key.json
-          gcloud auth activate-service-account --key-file ~/gcloud-service-key.json
-          gh-actions-scripts/upload_helm_chart_to_google_cloud.sh dist/keptn-installer/
+          mv "$ARTIFACTS" ./helm-charts/packages
+          cd ./helm-charts
+          helm repo index ./ --url https://charts.keptn.sh/ --merge ./index.yaml
+          git add --all
```

Comment:
This is mega dangerous. If for whatever reason there is a logfile with secrets or alike in this directory, we will leak it!
We should make sure that we add files by extension here.

Reply:
good catch, i will change it to the following:
```
git add *.tgz
git add index.yaml
```